test/MetadataUtility.Tests/TestHelpers/FixtureModel.cs

```diff
@@ -115,6 +90,34 @@ public TargetInformation ToTargetInformation(IFileSystem fileSystem)
             return ti;
         }
 
+        /// <summary>
+        /// Overwrite certain values in the recording.
+        /// Used for edge cases with some extractor.
+        /// </summary>
+        /// <param name="overwriteRecording">The FixtureModel containing the values to overwrite.</param>
+        public void ApplyOverwrites(FixtureModel overwriteRecording)
+        {
+            foreach (string overwrite in overwriteRecording.OverwriteProperties)
```

Comment:
Ok I just worked out why you explicitly listed properties together.

If you can't the yaml stream merging working (as per https://github.com/QutEcoacoustics/audio-analysis/blob/85ac7985bcddbd5bec0bbb9b83057106d1442b17/src/Acoustics.Shared/Yaml.cs#L52-L59)

Then I'd suggest you deserialize `Process ` as a Dictionary<string, object>`. You can then use graph traversal and type testing to add properties to the record.